### PR TITLE
Task/add instllation guide for ansible galaxy

### PR DESCRIPTION
**Pull request by harrisonchen201703**
- Add usage in README.md for both download from github and Ansible-Galaxy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +24,18 @@ There are no dependencies.
 
 ## Usage
 
-Below are sample usages for the playbook:
+Currently we have provided 2 ways to retrive this role:
```
Comment (ChristinaSmith123) [inline README.md]: 2 -> two

Comment (ChristinaSmith123) [reply README.md]: retrive -> retrieve

File: README.md [hunk 2]
```diff
@@ -24,9 +24,18 @@ There are no dependencies.
...
+ * From Ansible-Galaxy
```
Comment (ChristinaSmith123) [inline README.md]: From Ansible-Galaxy -> Download from

File: README.md [hunk 3]
```diff
@@ -24,9 +24,18 @@ There are no dependencies.
...
+ * Clone from this repo
+
+### Download From Ansible-Galaxy
```
Comment (ChristinaSmith123) [inline README.md]: Download From Ansible-Galaxy -> Download from Ansible-Galaxy.
Please use 'sentence case' for all headings. Example:

Correct: Download your file now
Incorrect: Download Your File Now

Even though you are using sentence case, remember to preserve capital letters on things that require a capital letter.
Correct: Download the Agent now
Incorrect: Download the agent now

File: README.md [hunk 4]
```diff
@@ -58,6 +67,55 @@ Below are sample usages for the playbook:
       force_reactivation: false
 ```
 
+### Download From this repo
```
Comment (ChristinaSmith123) [inline README.md]: Download From -> Download from

**Review by ChristinaSmith123 — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -58,6 +67,55 @@ Below are sample usages for the playbook:
...
+
+```
+project
+│   playbook.yaml
+└───roles
+    └───ansible
+        │   LICENSE
+        │   README.md
+        └───defaults
+        │
```
Comment (lovemew67) [inline README.md]: MINOR: Remove redundant bar to see more compact.

File: README.md [hunk 6]
```diff
@@ -58,6 +67,55 @@ Below are sample usages for the playbook:
...
+        └───meta
+        │
```
Comment (lovemew67) [inline README.md]: Ditto.

**Review by lovemew67 — CHANGES_REQUESTED** [review]
See comments

File: README.md (hunk 3 above)
Comment (harrisonchen201703) [reply README.md]: Thanks for reminding, i'll check with rest of the content.

File: README.md (hunk 1 above)
Comment (harrisonchen201703) [reply README.md]: Thanks, will update in next commit.

File: README.md (hunk 2 above)
Comment (harrisonchen201703) [reply README.md]: Thanks, will update in next commit.

File: README.md (hunk 5 above)
Comment (harrisonchen201703) [reply README.md]: Thanks, will update in next commit.

File: README.md (hunk 6 above)
Comment (harrisonchen201703) [reply README.md]: Thanks, will update in next commit.

File: README.md [hunk 7]
```diff
@@ -24,11 +24,11 @@ There are no dependencies.
...
 
-Currently we have provided 2 ways to retrive this role:
- * From Ansible-Galaxy
+Currently we have provided two ways to retrieve this role:
+ * Download from Ansible-Galaxy
  * Clone from this repo
 
-### Download From Ansible-Galaxy
+### Download from Ansible-Galaxy
  * Please visit our role in [ Ansible-Galaxy](https://galaxy.ansible.com/deep-security/deep-security-agent)
```
Comment (lovemew67) [inline README.md]: [Ansible-Galaxy]

**Review by lovemew67 — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -58,6 +67,53 @@ Below are sample usages for the playbook:
       force_reactivation: false
 ```
 
+### Download from this repo
```
Comment (ChristinaSmith123) [inline README.md]: Sorry I didn't catch this but I think this should say 'Clone from this repo' so that it matches the wording at the top.

**Review by ChristinaSmith123 — COMMENTED** [review]
Looks good!

**Review by ChristinaSmith123 — APPROVED** [review]

**Review by lovemew67 — APPROVED** [review]

**Pull request merged**